ml-conversational-analytic-tool/run.py

```diff
@@ -2,14 +2,30 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import argparse
+import tarfile
+import os
+
 from sklearn.model_selection import train_test_split
 
 from baseCNN import BaseCNN
 from baseLSTM import BaseLSTM
 from preProcessedDataset import PreProcessedDataset
 
-
-def run(annotated_filename, dataset_filename, outcome, encoding_type, model_type, padding):
+model_directory = 'models'
+model_version = '0001'
+
+def save_model(model, name):
+    if not os.path.exists(model_directory):
+        os.makedirs(model_directory)
+    file_path = model_directory + "/" + name
+    model.saveModel(name=file_path, version=model_version)
+    os.chdir(file_path)
+    tar = tarfile.open(name + "-" + model_version + ".tar.gz", "w:gz")
+    tar.add(model_version)
+    tar.close()
+    os.chdir("../../")
```

Comment:
nit: what about printing out the directory the model and the tar file are saved?